reg_tests/executeOpenfastLinearRegressionCase.py

```diff
@@ -100,11 +108,11 @@ def isclose(a, b, rel_tol=1e-09, abs_tol=0.0):
 
 # verify all the required directories exist
 if not os.path.isdir(rtest):
-    rtl.exitWithError("The test data directory, {}, does not exist. If you haven't already, run `git submodule update --init --recursive`".format(rtest))
+    exitWithError("The test data directory, {}, does not exist. If you haven't already, run `git submodule update --init --recursive`".format(rtest))
```

Comment:
Not using the `rtl` library version anymore?

Reply:
I'm using the rtl version, but I just wrapped it into a function called exitWithError above in the script, so that the casename is written before the error.

Reply:
I missed that bit.  Sounds good!